### PR TITLE
[test-only] personal space is deleted after a user is deleted test

### PR DESCRIPTION
**Pull request by ScharfViktor**
- [x] checking that no personal space after deleting user
- [x]  checking that no share after deleting user

### CODE REVIEW

File: tests/acceptance/features/apiGraph/deleteUser.feature [hunk 1]
```diff
@@ -143,3 +143,27 @@ Feature: delete user
...
+
+  
+  Scenario: personal space is deleted automatically when the user is deleted
+    Given the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And user "Brian" has been created with default attributes and without skeleton files
+    When the user "Alice" deletes a user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    When user "Alice" lists all spaces via the GraphApi with query "$filter=driveType eq 'personal'"
+    Then the json responded should not contain a space with name "Brian Murphy"
+    And as "Alice" folder "Shares/new" should not exist
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/deleteUser.feature:155]: Maybe by mistake. The scenario should check only the personal space, no shares
```suggestion
```

File: tests/acceptance/features/apiGraph/deleteUser.feature [hunk 2]
```diff
@@ -143,3 +143,27 @@ Feature: delete user
...
+
+
+  Scenario: accepted share is deleted automatically when the user is deleted
+    Given the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Brian" has created a folder "new" in space "Brian Murphy"
+    Given user "Brian" has created a share inside of space "Brian Murphy" with settings:
+      | path       | new                   |
+      | shareWith  | Alice                    |
+      | role       | viewer                   |
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/deleteUser.feature:165]: table format
```suggestion
      | path      | new    |
      | shareWith | Alice  |
      | role      | viewer |
```

**Review by saw-jan — APPROVED** [review]
LGTM

File: tests/acceptance/features/apiGraph/deleteUser.feature [hunk 3]
```diff
@@ -143,3 +143,26 @@ Feature: delete user
...
+    Then the HTTP status code should be "204"
+    When user "Alice" lists all spaces via the GraphApi with query "$filter=driveType eq 'personal'"
+    Then the json responded should not contain a space with name "Brian Murphy"
+
+
+  Scenario: accepted share is deleted automatically when the user is deleted
+    Given the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Brian" has created a folder "new" in space "Brian Murphy"
+    Given user "Brian" has created a share inside of space "Brian Murphy" with settings:
```
Comment (SagarGi) [inline tests/acceptance/features/apiGraph/deleteUser.feature:161]:
```suggestion
    And user "Brian" has created a share inside of space "Brian Murphy" with settings:
```

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Pull request merged**